File: FizzBuzz/fizzbuzz.c
#include <stdio.h>
int i;

int main(){
    // Create a loop that counts to 50
    for(i=1;i<=50;i++){
        
    
        // Replace #'s divisible by 3 and 5 with "FizzBuzz"
        if(i%3==0 && i%5==0){
            printf("FizzBuzz\n");
        // Replace #'s divisible by 3 with "Fizz"
        }else if (i%3==0){
            printf("Fizz")
        // Replace #'s divisible by 3 with "Fizz"

        // Replace #'s divisible by 5 with "Buzz"
        }else
        // Print all other numbers

        printf("%d\n", i);
        }
    }
    return 0;
    
}

File: FizzBuzz/fizzbuzz.py
i=1.
while(i<=50):
    i=i+1
    for i in range(51):
        if i%3==0 and i%5==0:
            print("FizzBuzz")
        elif i%3==0:
            print("Fizz")
        elif i%5==0:
            print("Buzz")
        else:
            print(i)
    break